services/messages/messages_file.py

```diff
@@ -0,0 +1,38 @@
+import json
+import time
+
+from settings import FileConf, SLEEP_TIME
+from util.messages import send_message
+
+
+class MessagesFile(object):
```

Comment:
Restructure MessageFile and utils.messages into classes, everything inside services